### PR TITLE
Fix/162

### PR DESCRIPTION
**Pull request by bohdanyurov-gl**

### CODE REVIEW

File: dm/templates/iam_member/iam_member.py [hunk 1]
```diff
@@ -13,28 +13,58 @@
...
 
 def generate_config(context):
     """ Entry point for the deployment resources. """
 
-    project_id = context.properties.get('projectId', context.env['project'])
+    folder_id = context.properties.get('folderId')
+    org_id = context.properties.get('organizationId')
+    project_id = context.properties.get('projectId')
+    if not folder_id and not org_id:
+        project_id = context.env['project']
```
Comment (ocsig) [inline dm/templates/iam_member/iam_member.py]: This means `context.properties.get('projectId')` is never used. Line 25-26 should be removed.
Line 24 should be:
`    project_id = context.properties.get('projectId', context.env['project'])`

File: dm/templates/iam_member/iam_member.py.schema [hunk 2]
```diff
@@ -35,40 +39,52 @@ required:
   - roles
```
Comment (ocsig) [inline dm/templates/iam_member/iam_member.py.schema]: folredID, OrgID, projectID shall not be used at the same time. Please specify this so there will be no blind overwrite.

Comment (bohdanyurov-gl) [reply dm/templates/iam_member/iam_member.py.schema]: Done.

Comment (bohdanyurov-gl) [reply dm/templates/iam_member/iam_member.py.schema]: This is not a problem, you can use all three if you want to. This template supports it.

File: dm/templates/iam_member/iam_member.py (hunk 1 above)
Comment (bohdanyurov-gl) [reply dm/templates/iam_member/iam_member.py]: Fixed.

**Pull request merged**